### PR TITLE
Temporary cleanup missing/unready/need-refactoring elements

### PR DESCRIPTION
**Pull request by nervo**

### CODE REVIEW

File: README.app.md [hunk 1]
```diff
@@ -40,9 +40,6 @@ Run test
 
 Tests files should be in the `tests/` directory and under the namespace `Tests\\`
 
-* *Supervisor*: http://{{ app_host }}.dev:9001
```
Comment (Tom32i) [inline README.app.md]: No more supervisor by default?

File: ansible/group_vars/all.yml [hunk 2]
```diff
@@ -194,7 +141,7 @@ elao_phpmyadmin_config_template: config/{{ _env }}.inc.php.j2
 # Php #
 #######
 
-elao_php_version: 5
+elao_php_version: 7.0
```
Comment (Tom32i) [inline ansible/group_vars/all.yml]: :clap:

File: ansible/group_vars/all.yml [hunk 3]
```diff
@@ -204,20 +151,16 @@ elao_php_fpm_pools:
     template: fpm_pools/www_{{ _env }}.conf.j2
 
 elao_php_extensions:
-  # Dev
-  - xdebug
```
Comment (Tom32i) [inline ansible/group_vars/all.yml]: No more xdebug? How so?

File: README.app.md (hunk 1 above)
Comment (nervo) [inline README.app.md]: Fortunately, yes :)
The fact is the role is still not ready for jessie. In few days, i will add it back again.

File: ansible/group_vars/all.yml (hunk 3 above)
Comment (nervo) [inline ansible/group_vars/all.yml]: The extension is not upstream ready for php7. I will add it back again when it's ready.

**Pull request merged**